features/travel_advice.feature

```diff
@@ -23,3 +23,8 @@ Feature: Travel Advice
       | Path                                   |
       | /foreign-travel-advice.atom            |
       | /foreign-travel-advice/luxembourg.atom |
+
+  @normal
+  Scenario: Country feed should contain correct website root
+    When I visit "/foreign-travel-advice/ireland.atom"
+    Then I should see the contents of environment variable "EXPECTED_GOVUK_WEBSITE_ROOT" followed by "/foreign-travel-advice/ireland"
```

Comment:
in this case dont we want to make sure that the atom id is correct since that is what triggered the bug. 
```
<id>https://www.gov.uk/foreign-travel-advice/ireland#2015-09-11T11:56:59+01:00</id>
```